schemas/digital-outcomes-and-services-2.declaration.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "allOf": [
+        {"$ref": "#/definitions/notDefiniteFail"},
+        {
+            "properties": {
+                "taxEvasion": {"enum": [false]},
+                "environmentalSocialLabourLaw": {"enum": [false]},
+                "bankrupt": {"enum": [false]},
+                "graveProfessionalMisconduct": {"enum": [false]},
+                "distortingCompetition": {"enum": [false]},
+                "conflictOfInterest": {"enum": [false]},
+                "distortedCompetition": {"enum": [false]},
+                "significantOrPersistentDeficiencies": {"enum": [false]},
+                "seriousMisrepresentation": {"enum": [false]},
+                "witheldSupportingDocuments": {"enum": [false]},
+                "influencedContractingAuthority": {"enum": [false]},
+                "confidentialInformation": {"enum": [false]},
+                "misleadingInformation": {"enum": [false]},
+                "unspentTaxConvictions": {"enum": [false]},
+                "GAAR": {"enum": [false]}
+            }
+        }
+    ],
+    "definitions": {
+        "notDefiniteFail": {
```

Comment:
I might be misunderstanding, but I think `notDefiniteFail` is the wrong name here altogether (although the script is doing the right thing with them) - if these ones aren't right then it *is* a definite fail.  It's the ones above (taxEvasion...GAAR) where it's not a definite fail if the schema isn't matched.